app/containers/SignUp/saga.js

```diff
@@ -356,6 +364,49 @@ export function* signUpWithScatterWorker({ val }) {
   }
 }
 
+export function* signUpWithKeycatWorker({ val }) {
```

Comment:
This code is duplication of `signUpWithScatterWorker`. Code has to be reused.

Reply:
fixed